### PR TITLE
Fix: updated webpack to avoid self in ssr and added checks for window object

### PR DESCRIPTION
**Pull request by usmaniqbal998**
- Fixed webpack build to not generate "self" because its available in browser only
- for browser apis added checks for window object available
- version updated

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -22,27 +22,27 @@ export type {
   SliderType,
   FilterKey,
   FilterBaseDataType,
+  Range,
```
Comment (birkanalp) [inline src/index.ts:25]: @usmaniqbal998 Can you use Range component when publish in "export type".?

Comment (usmaniqbal998) [reply src/index.ts:25]: its working fine on local but yes this should be corrected still

Comment (usmaniqbal998) [reply src/index.ts:25]: it works fine but actually breaks when i move it other way around

Comment (usmaniqbal998) [reply src/index.ts]: this is fixed

**Review by birkanalp — APPROVED** [review]

**Pull request merged**